src/main/java/lotto/LottoApplication.java

```diff
@@ -0,0 +1,26 @@
+package lotto;
+
+
+import java.util.List;
+
+import lotto.service.LottoGame;
+import lotto.strategy.LottoGenerator;
+
+import static lotto.view.Input.inputLastWeeksWinningNumbers;
+import static lotto.view.Input.inputPurchaseAmount;
+import static lotto.view.Output.getPurchasedLottoCount;
+import static lotto.view.Output.printGeneratedLottoNumbers;
+
+public class LottoApplication {
+
+
+    public static void main(String[] args) {
+        int ticketCount = inputPurchaseAmount();
+        getPurchasedLottoCount(ticketCount);
+        List<Integer> lottoNumbers = LottoGenerator.generateLottoNumbers(ticketCount);
```

Comment:
```suggestion
        int money = inputPurchase();
        List<Lotto> lottos = LottoGenerator.generateLottoNumbers(money);
        printGeneratedLottoNumbers(lottos); // 로또 개수와 함께 로또 리스트 출력
```
개수 출력은 로또 리스트를 출력하면서 할 수 있을 것 같습니다.